src/core/cfg.h

```diff
@@ -43,6 +44,12 @@ class CFGBlock {
     CFG* cfg;
 
 public:
+    // Baseline JIT helper fields:
+    // contains address to the start of the code of this basic block
+    void* code;
+    // contains the address of the entry function
+    std::pair<CFGBlock*, Box*>(*entry_code)(void* interpeter, CFGBlock* block);
```

Comment:
yeah having these here isn't great, but I think it's ok for now.  I haven't read through all the code yet, but maybe we could some other data structure whose fields are `CFGBlock* block; void* code; ... *entry_code;`, and then add them to the CLFunction?